### PR TITLE
Updated to run on FreeBSD 10.1

### PR DESCRIPTION
**Pull request by RussellHaley**
Updated the Makefile.am to run mozroots and execute mono using explicit syntax. Also updated the configure.sh to set SHELL env. Not sure how best to do that without breaking the script for Linux.

### CODE REVIEW

File: main/Makefile.am [hunk 1]
```diff
@@ -20,7 +20,8 @@ clean: clean-local
 clean-local: sln_clean
 
 restore-packages:
-	nuget restore
+	mozroots --import --sync
+	mono /usr/home/rhaley/git/monodevelop/main/external/nuget-binary/NuGet.exe restore
```
Comment (mhutch) [inline main/Makefile.am]: This has a local path, it'll only work on your machine.

File: main/Makefile.am [hunk 2]
```diff
@@ -20,7 +20,8 @@ clean: clean-local
 clean-local: sln_clean
 
 restore-packages:
-	nuget restore
+	mozroots --import --sync
```
Comment (mhutch) [inline main/Makefile.am]: This should be done when installing the Mono package.

File: scripts/configure.sh [hunk 3]
```diff
@@ -1,3 +1,5 @@
+env SHELL=/usr/local/bin/bash
```
Comment (mhutch) [inline scripts/configure.sh]: Most systems don't have bash installed to /usr/local

File: main/Makefile.am (hunk 2 above)
Comment (RussellHaley) [inline main/Makefile.am]: nuget restore wouldn't run until I ran mozroots, which caused the build to fail.

File: main/Makefile.am (hunk 1 above)
Comment (RussellHaley) [inline main/Makefile.am]: quite. I'll re-submit.

File: scripts/configure.sh (hunk 3 above)
Comment (RussellHaley) [inline scripts/configure.sh]: I'll look at having this added to the platform specific Makefile (FreeBSD ports). Is there a cross platform/generic command for ensuring bash is the correct shell?

**Comment by RussellHaley** [comment]
Great, thanks for the input Michael!

**Pull request closed without merging**